eZ/Publish/Core/Persistence/Legacy/Content/Location/Gateway/DoctrineDatabase.php

```diff
@@ -631,6 +673,30 @@ function ($pathString) use ($query, $handler) {
         $query->prepare()->execute();
     }
 
+    private function buildHiddenSubtreeQuery(): QueryBuilder
+    {
+        $query = $this->connection->createQueryBuilder();
+        $expr = $query->expr();
+        $query
+            ->select('node_id')
+            ->from('ezcontentobject_tree', 't')
+            ->join('t', 'ezcontentobject', 'c', 't.contentobject_id = c.id')
+            ->where(
+                $expr->or(
+                    $expr->eq(
+                        't.is_hidden',
+                        $query->createPositionalParameter(true, ParameterType::BOOLEAN)
```

Comment:
This one is actually integer, only `c.is_hidden` is boolean.
```suggestion
                        $query->createPositionalParameter(true, ParameterType::INTEGER)
```

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/3142/commits/15d0c46606e8dbc6df39b58eaf98706a05174d81